addon/background.js

```diff
@@ -49,3 +49,42 @@ browser.tabs.onUpdated.addListener((tabId, changeInfo, tabInfo) => {
             });
     }
 });
+
+async function freeze_active_tab(tab) {
+    // TODO: Is this try-catch unnecessary since it's being called below in a try-catch?
+    try {
+        await browser.tabs.executeScript(
+            tab.id,
+            {file: '/contentScript.js'}
```

Comment:
I don't *think* this should be necessary, since the manifest.json already makes the content script execute on every page:

```js
    "content_scripts": [{
        "matches": ["<all_urls>"],
        "js": ["utils.js", "contentScript.js", "simmer.js"]
    }],
```

Reply:
You are correct! Do you know why we need to do it [here](https://github.com/mozilla/fathom-fox/blob/550e193bc2274602050b92a29961a4d3a3fd9922/addon/corpus.js#L67) then? This is what made me think I needed to include it.

Reply:
Hmm, I do not. We should try without.